### PR TITLE
Velbus config parameters

### PR DESCRIPTION
**Pull request by cereal2nd**
## Proposed change

Some changes in the velbus docs, make it more clear what the port parameter can be
Renmae services to actions

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Added note about pushbutton LEDs being disabled by default and how to enable them.
	- Updated section titles from "Services" to "Actions" for clarity.
	- Restructured port string format section to specify formats for different connection types.
	- Introduced new sections for removing the Velbus integration and configuring the VMB7IN sensor for energy monitoring.
	- Enhanced organization and terminology throughout the documentation.

### CODE REVIEW

**Comment by cereal2nd** [comment]
@coderabbitai review

File: source/_integrations/velbus.markdown [hunk 1]
```diff
@@ -31,18 +31,33 @@ ha_integration_type: hub
 
 The **Velbus** {% term integration %} is used to control [Velbus](https://www.velbus.eu/?lang=en) modules. It supports the Velbus USB, Velbus serial and a TCP/IP gateway.
 
+<div class='note'>
```
Comment (frenck) [inline source/_integrations/velbus.markdown]:
```suggestion
{% note %}
```

File: source/_integrations/velbus.markdown [hunk 2]
```diff
@@ -31,18 +31,33 @@ ha_integration_type: hub
...
 The pushbutton LEDs of input modules are disabled by default. These can be enabled from the `Devices` panel in the `Configuration` page of the web interface.
+</div>
```
Comment (frenck) [inline source/_integrations/velbus.markdown]:
```suggestion
{% endnote %}
```

File: source/_integrations/velbus.markdown [hunk 3]
```diff
@@ -31,18 +31,33 @@ ha_integration_type: hub
 
 The **Velbus** {% term integration %} is used to control [Velbus](https://www.velbus.eu/?lang=en) modules. It supports the Velbus USB, Velbus serial and a TCP/IP gateway.
 
+<div class='note'>
 The pushbutton LEDs of input modules are disabled by default. These can be enabled from the `Devices` panel in the `Configuration` page of the web interface.
```
Comment (frenck) [inline source/_integrations/velbus.markdown]: I think we need to move this further down.

Serving a note before the user has even read the configuration documentation, is quite odd :)

File: source/_integrations/velbus.markdown [hunk 4]
```diff
@@ -31,18 +31,33 @@ ha_integration_type: hub
...
+- For Signum devices without authentication: `tls://192.168.1.9:27015`
+- For Signum devices with authentication: `tls://password@192.168.1.9:27015`
+
+### Removing the integration
+
+The Velbus integration and its entities can be removed by following these steps:
+
+1. Navigate to Settings > Devices & Services
+2. Find the Velbus integration and click on it
+3. Click the delete button to remove the Velbus config entry
```
Comment (frenck) [inline source/_integrations/velbus.markdown]: We have a snippet for this that should be included. See for example, the Twente Milieu documentation.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/velbus.markdown [hunk 5]
```diff
@@ -31,18 +31,32 @@ ha_integration_type: hub
...
+- For Signum devices without authentication: `tls://192.168.1.9:27015`
+- For Signum devices with authentication: `tls://password@192.168.1.9:27015`
+
+### Removing the integration
+
+The Velbus integration and its entities can be removed by following these steps:
+
+{% include integrations/remove_device_service.md %}
+
+Note: Removing the integration will delete all Velbus devices and their history from Home Assistant.
```
Comment (frenck) [inline source/_integrations/velbus.markdown]: Removal should be at almost the very end of the documentation page.

File: source/_integrations/velbus.markdown [hunk 6]
```diff
@@ -31,18 +31,32 @@ ha_integration_type: hub
...
+
+{% note %}
+The pushbutton LEDs of input modules are disabled by default. These can be enabled from the `Devices` panel in the `Configuration` page of the web interface.
```
Comment (frenck) [inline source/_integrations/velbus.markdown]:
```suggestion
The pushbutton LEDs of input modules are disabled by default. These can be enabled from the **Devices** panel in the **Configuration** page of the web interface.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @cereal2nd 👍 

../Frenck

**Pull request merged**